docs/reference/github/workflow/branches.md

```diff
@@ -0,0 +1,148 @@
+# Developing on Branches
+
+We use branching to diverge from the main line of development and work on issues without directly
+modifying the main line. This ensures that our codebase is robust and avoids merge conflicts
+when multiple people are developing at the same time. For a quick rundown on branching in git,
+consult the official [git documentation](https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell).
+
+## Creating a branch
+
+There are three possible methods to create a branch for your next issue:
```

Comment:
I think that we should still keep this information here because this allows developers to create their own local branches where they can branch off ideas. I agree that we should only branch from `main` for branches **related to issues**. I think that beyond this, we should have the freedom to do whatever we want in our local workspace as long as it does not interfere with the remote repository.

I'll add a note that when creating branches for issues, we should do it from `main`, but I still think that we should keep the information.

Reply:
If we are only documenting what you need for sailbot to get started I don't think the other options are necessary, but as long as the sailbot convention is front and center I 'm good

Reply:
Good point, I should probably simplify this part. I will address this.